Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -946,7 +946,7 @@ public AndConstraint<TAssertions> ContainInOrder(params T[] expected)
         }
 
         /// <summary>
-        /// Asserts that the current collection only contains items that are assignable to the type <typeparamref name="TExpectation" />.
+        /// Asserts that the current collection contains items that are assignable to the type <typeparamref name="TExpectation" />.
```

Comment:
🔧 I would rephrase the middle of the message as "to contain at least one element...."

Reply:
Done in 3e7255089ab330db576c5c91d5a63daf8f10611a.